srcpkgs/noto-fonts-ttf/template

```diff
@@ -1,25 +1,20 @@
 # Template file for 'noto-fonts-ttf'
 pkgname=noto-fonts-ttf
-version=20220810
+reverts="20220810_1 20190926_2 20190926_1 20181202_1 20181106_1"
+version=23.2.1
 revision=1
-_githash=2725c70baa8b0176c7577093ba1fc6179aa79478
 depends="font-util"
 short_desc="Google Noto TTF Fonts"
 maintainer="classabbyamp <void@placeviolette.net>"
-license="OFL-1.1"
-homepage="https://www.google.com/get/noto/"
-# XXX: change to https://github.com/notofonts/notofonts.github.io/tree/main/fonts when migration is complete
-# also see https://github.com/notofonts/noto-fonts/issues/2388
-distfiles="https://github.com/notofonts/noto-fonts/archive/${_githash}.tar.gz"
-checksum=c1238ca0fc1e954ee2be444cafd53c033b13fa93f8a1e88f7137e9e028e6891e
+license="Apache-2.0"
```

Comment:
the fonts are OFL (https://github.com/notofonts/notofonts.github.io/blob/main/fonts/LICENSE), the website is apache

```suggestion
license=OFL-1.1"
```

Reply:
Ah, okay. Should I put the vlicense back? xlint doesn't complaint tho.

Reply:
i think so? https://github.com/notofonts/notofonts.github.io/blob/main/fonts/LICENSE#L57-L62